integration_tests/utils.js

```diff
@@ -148,7 +148,7 @@ const extractSummary = (stdout: string) => {
 
   const rest = cleanupStackTrace(
     // remove all timestamps
-    stdout.slice(0, -match[0].length).replace(/\s*\(.*ms\)/gm, ''),
+    stdout.slice(0, -match[0].length).replace(/\s*\(\d*\.?\d+m?s\)/gm, ''),
```

Comment:
just adding `m?s` made a single other test fail.

![image](https://user-images.githubusercontent.com/1404810/34155784-03a07576-e4bb-11e7-9b5c-6a87e95344c1.png)

Reply:
Maybe worth to add a `$` so we match only the end of the string?

Reply:
Yeah, that probably would have fixed it. added it anyways 🙂